doc/news/_preparation_next_release.md

```diff
@@ -125,6 +125,26 @@ you up to date with the multi-language support provided by Elektra.
 
 ## Tools
 
+- `KDB_EXEC_PATH`, which can be used to add further external tools to `kdb`, now supports `:` to separate paths.
+  `kdb list-tools` and `run_env` were improved to take advantage of this. _(Markus Raab)_
+- Checks for `kdbCommit` have been added to [kdb plugin-check](../help/kdb-plugin-check.md). _(Vid Leskovar)_
+- add PID file config setting for kdb-run-rest-frontend _(Markus Raab)_
```

Comment:
Are these changes really part of this pr or is this a merge error? On that note, please rebase instead of merge in future to avoid many merge commits (mentioned in .github/CONTRIBUTING.md).